samples/matter/lock/Kconfig

```diff
@@ -6,17 +6,42 @@
 
 mainmenu "Matter Lock sample application"
 
-config LOCK_NUM_USERS
+config LOCK_MAX_NUM_USERS
 	int "Maximum number of users supported by lock"
 	default 10
 
-config LOCK_NUM_CREDENTIALS
-	int "Maximum number of credentials supported by lock"
-	default 20
-
 config LOCK_NUM_CREDENTIALS_PER_USER
-	int "Maximum number of credentials per user supported by lock"
-	default 3
+	int "Maximum total number of credentials per one user supported by lock"
+	default 4
+
+config LOCK_NUM_CREDENTIALS_TYPE
+	int "Maximum number of credentials of given type"
+	default 15
+
+config LOCK_MAX_CREDENTIAL_LENGTH
+	int "Maximum length of the single credential"
+	default 10
+
+config LOCK_ENABLE_DEBUG
+	bool "Enable debug features to print users and credentials"
+	help
+	  Debug features allows to checking and printing current users, their state, and
+	  associated credentials. Use these with shell commands combinations to read saved credentials
+	  or users, and manage them (add, clear, read).
+
+if LOCK_ENABLE_DEBUG
+
+config SHELL_CMD_BUFF_SIZE
+	default 396
```

Comment:
Just out of my curiosity, why 396? Maybe let's add some comment to not wonder about it after some time.

Reply:
Probably I set this value a long time ago, and it describes the maximum size of the very specific credential type. :)

Reply:
It was more specific to the project Fell and dependent on the LOCK_MAX_CREDENTIAL_LENGTH. I guess we may safely decrease it for the purposes of the lock app having in mind that we have LOCK_MAX_CREDENTIAL_LENGTH equal to 10.